filebeat/harvester/log.go

```diff
@@ -36,6 +37,8 @@ func NewHarvester(
 		encoding:         encoding,
 		backoff:          prospectorCfg.Harvester.BackoffDuration,
 	}
+	h.ExcludeLinesRegexp = InitRegexp(h.ProspectorConfig.ExcludeLines)
+	h.IncludeLinesRegexp = InitRegexp(h.ProspectorConfig.IncludeLines)
```

Comment:
maybe we can compile these very very early in prospector or crawler during startup. Failing to compile the regular expressions during startup, we can kill the process almost immediately. Trying to handle error in harvester is kinda late. We still want to run harvester if filters are not properly setup?